pull_request_template.md

```diff
@@ -2,4 +2,7 @@
 Please provide a description of your changes
 
 **Testing**
-How did you test this change?
+
+- [ ] Test authenticating with sas server
```

Comment:
@scnwwu @clangsmith Are there other basic things that should be tested when creating a new pull request?

Reply:
I'm not sure what's the template should include. I guess the required tests depend on the change scope.
If a change is LSP related, one should probably test the LSP features such as syntax highlighting, auto-complete, etc.
If a change is environment related, one should probably test remote mode or web worker mode.
If a change is related to run code, one should probably test some specific code, like long runs, syntax check mode, ods, etc. Or different kinds of back-end.
Regarding baseline (regression) tests, I guess they'd better be covered by unit tests and automation.

Reply:
> Regarding baseline (regression) tests, I guess they'd better be covered by unit tests and automation.

Strongly Agree here. Ideally the test automation would catch regressions. A condition of code review should also be that relevant covering unit tests are written.